crowbar_framework/spec/controllers/deploy_queue_controller_spec.rb

```diff
@@ -41,12 +41,12 @@
     describe "with existing nodes" do
       before do
         # Simulate the expansion to a node whose look up we can fake
-        ServiceObject.stubs(:expand_nodes_for_all).returns([["testing.crowbar.com"],[]])
+        allow(ServiceObject).to receive(:expand_nodes_for_all).and_return([["testing.crowbar.com"],[]])
```

Comment:
Metrics/LineLength: Line is too long. [103/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SpaceAfterComma: Space missing after comma. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)